apps/condo/domains/banking/schema/BankAccount.test.js

```diff
@@ -62,6 +73,30 @@ describe('BankAccount', () => {
                 expect(bankAccount.currencyCode).toEqual('RUB')
             })
 
+            test('service can if organization have context', async () => {
+                const [organization] = await createTestOrganization(adminClient)
+                await createTestBankIntegrationOrganizationContext(adminClient, SBBOLBankIntegration, organization)
+
+                const [bankAccount] = await createTestBankAccount(serviceClient, organization)
+
+                expect(bankAccount.organization.id).toEqual(organization.id)
+                expect(bankAccount.tin).toBeDefined()
+                expect(bankAccount.country).toEqual('ru')
+                expect(bankAccount.routingNumber).toBeDefined()
+                expect(bankAccount.number).toBeDefined()
+                expect(bankAccount.currencyCode).toEqual('RUB')
+            })
+
+            test('service can\'t if organization without context', async () => {
+                const [organization] = await createTestOrganization(adminClient)
+                try {
+                    await createTestBankAccount(serviceClient, organization)
+                } catch (error) {
+                    expect(error.message).toContain('You do not have access to this resource')
+                }
```

Comment:
It looks like you can use `expectToThrowAccessDeniedErrorToObj` here, but due to specific of this test, which triggers `false` not in access of BankAccount, but in access of Organization, — we get nested error, not related to access of BankAccount.

Could you please either write a comment about this or to write additional test util, that checks for errors, located one level deeper. Something like `expectToThrowAccessDeniedNestedErrorToObj`.